app/controllers/report_controller/reports.rb

```diff
@@ -373,6 +373,7 @@ def check_tabs
         elsif @edit[:new][:model] == "Chargeback"
           unless @edit[:new][:cb_show_typ] &&
                  ((@edit[:new][:cb_show_typ] == "owner" && @edit[:new][:cb_owner_id]) ||
+                  (@edit[:new][:cb_show_typ] == "tenant" && @edit[:new][:cb_tenant_id]) ||
```

Comment:
Ok @lpichler It's fine. I am working on it. I have 2 proposal that you can check here [https://tree.taiga.io/project/sergioocon-charging-in-manageiq/task/40](url) but **Rubocop** throw a complexity errors  :(

1)
```Ruby
  # Check if chargeback field is valid
  VALID_SHOW_TYP_FIELDS = %w(owner tenant tag).freeze
  def valid_chargeback_fields
    if @edit[:new][:cb_show_typ] && VALID_SHOW_TYP_FIELDS.include?(@edit[:new][:cb_show_typ])
      case @edit[:new][:cb_show_typ]
      when "owner"
        return true if @edit[:new][:cb_owner_id]
      when "tenant"
        return true if @edit[:new][:cb_tenant_id]
      when "tag"
        return true if @edit[:new][:cb_tag_cat] && @edit[:new][:cb_tag_value]
      end
    end
    false
  end
```
2)
```Ruby
  VALID_SHOW_TYP_FIELDS = %w(owner tenant tag).freeze
  def valid_chargeback_fields
    if @edit[:new][:cb_show_typ] && VALID_SHOW_TYP_FIELDS.include?(@edit[:new][:cb_show_typ])
      case @edit[:new][:cb_show_typ]
      when "owner" then return true if @edit[:new][:cb_owner_id]
      when "tenant" then return true if @edit[:new][:cb_tenant_id]
      when "tag" then return true if @edit[:new][:cb_tag_cat] && @edit[:new][:cb_tag_value]
      end
    end
    false
  end
```